ts/BUILD.typescript

```diff
@@ -0,0 +1,42 @@
+load("@aspect_bazel_lib//lib:copy_directory.bzl", "copy_directory")
+load("@aspect_rules_js//js:nodejs_package.bzl", "nodejs_package")
+load("@aspect_rules_js//js:nodejs_binary.bzl", "nodejs_binary")
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+
+copy_directory(
+    name = "_npm_typescript",
+    src = "package",
+    out = "typescript",
+)
+
+nodejs_package(
+    name = "npm_typescript",
+    src = "_npm_typescript",
+    package_name = "typescript",
+    visibility = ["//visibility:public"],
+    deps = [],
+)
+
+write_file(
```

Comment:
Why is this required?

Reply:
the new `nodejs_binary` here doesn't support `$(rootpath)` expansions so we can't point it directly at the `external/npm_typescript/.../tsc.js` file.

Could be removed in the future without change to user API